src/app/modules/convai-checker/perspective-status.component.ts

```diff
@@ -451,11 +436,9 @@ export class PerspectiveStatusComponent implements OnChanges, OnInit, AfterViewI
     // Use Math.floor because control points have to be integers.
     const gradientPoints = [
       Math.floor(
-        gradientPointCount * (
-          this.scoreThresholds[0] + FIRST_GRADIENT_RATIO * (
-            this.scoreThresholds[1] - this.scoreThresholds[0]))),
-      Math.floor(gradientPointCount * this.scoreThresholds[1]),
-      Math.floor(gradientPointCount * this.scoreThresholds[2])
+        gradientPointCount * (FIRST_GRADIENT_RATIO * this.scoreThresholds[0])),
```

Comment:
nit: It might be worth pulling `this.scoreThresholds[0]` and `this.scoreThresholds[1]` into variables. It gets a little tricky to remember what they mean when looking at this code.

Reply:
Done.